scripts/install-maria.sh

```diff
@@ -53,3 +55,5 @@ mysql --user="root" --password="secret" -e "GRANT ALL ON *.* TO 'homestead'@'0.0
 mysql --user="root" --password="secret" -e "GRANT ALL ON *.* TO 'homestead'@'%' IDENTIFIED BY 'secret' WITH GRANT OPTION;"
 mysql --user="root" --password="secret" -e "FLUSH PRIVILEGES;"
 service mysql restart
+
+touch /home/vagrant/.maria
```

Comment:
This isn't a terrible idea, but we have root access when we're running these, lets drop this somewhere outside of `/home/vagrant`. This would be useful as it's own PR outside of the `set -e` as well.